transaction.go

```diff
@@ -766,6 +767,11 @@ func (tx *Transaction) ProcessResponseBody() (*types.Interruption, error) {
 	reader = io.LimitReader(reader, tx.Waf.ResponseBodyLimit)
 	buf := new(strings.Builder)
 	length, _ := io.Copy(buf, reader)
+	// TODO should we handle this case better?
```

Comment:
What mean in this?

Reply:
I thought there was no way to know the body size if we only read a limited part but I forgot we have bodyBuffer.Size(), I must fix this

Reply:
fixed